src/Generated/ChatClientGetChatCompletionsAsyncCollectionResult.cs

```diff
@@ -0,0 +1,67 @@
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+
+namespace OpenAI.Chat
+{
+    internal partial class ChatClientGetChatCompletionsAsyncCollectionResult : AsyncCollectionResult
```

Comment:
@jorgerangel-msft @JoshLove-msft: What do you think about generating these internal pagination classes under `Generated/Internal` instead of `Generated` ?

Reply:
Internal is meant for internal helper types that are consistent across libraries. They aren't meant for library specific types that happen to be internal. We can consider renaming this folder, but I wouldn't put these collection types in the same folder as the Internal helper types.